pep-0649.rst

```diff
@@ -437,11 +439,11 @@ difficult to determine the correct globals and locals dicts to
 give to  ``eval`` to properly evaluate a stringized annotation.
 Even worse, in some situations it may simply be infeasible.
 
-For example, some libraries (e.g. TypedDict, dataclass) wrap a user
-class, then merge all the annotations from all that class's base
-classes together into one cumulative annotations dict.  If those
-annotations were stringized, calling ``eval`` on them later may
-not work properly, because the globals dictionary used for the
+For example, some libraries (e.g. ``typing.TypedDict``, dataclasses)
```

Comment:
Can you also please change dataclasses to :mod:`dataclasses` here, and throughout the document?

Reply:
Done!